.github/workflows/tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Tests
+
+on:
+  pull_request:
+    branches: [ main ]
+
+env:
+  JLM_ROOT_DIR: /tmp/jlm
+  CIRCT_INSTALL_DIR: /tmp/circt
+
+jobs:
+
+  hls-test-suite:
+    runs-on: ubuntu-22.04
+    if: contains(github.event.pull_request.title, '[DisableCI]') == false
+    steps:
+    - uses: actions/checkout@v4
+    - name: "Install packages"
+      uses: ./.github/actions/InstallPackages
+
+    - name: "Clone jlm"
+      run: git clone https://github.com/phate/jlm.git $JLM_ROOT_DIR
+      shell: bash
+
+    - name: "Build CIRCT and configure and build jlm"
+      run: |
+        cd $JLM_ROOT_DIR && \
+        ./scripts/build-circt.sh --install-path $CIRCT_INSTALL_DIR && \
+        ./configure.sh --enable-hls $CIRCT_INSTALL_DIR --target release --enable-asserts CXX=clang++ && \
+        make -j `nproc` -O all
+      shell: bash
+
+    - name: "MLIR libs"
+      run: |
+        if ! [ -f /usr/lib/x86_64-linux-gnu/libMLIR.so ]; then
+          sudo ln -s /usr/lib/llvm-17/lib/libMLIR.so.17 /usr/lib/x86_64-linux-gnu/
+          sudo ln -s /usr/lib/llvm-17/lib/libMLIR.so.17 /usr/lib/x86_64-linux-gnu/libMLIR.so
+        fi
+      shell: bash
```

Comment:
Could we introduce a environment variable `LLMV_VERSION` where we set the LLVM version and then use it here as well as in the InstallPackages script? In other words, one place to rule it all such that we only have to change this one place when we update LLVM in the future.

Reply:
These shouldn't have to be here and are annoying. It runs on my machine locally, but for some reason the LD_LIBRARY_PATH in the makefile gets ignored in the CI.

Reply:
Added 17 as vairable